features/polyTalkPreview/src/locales/de/preview.json

```diff
@@ -4,12 +4,12 @@
     "title": "Für Sie und Ihre Privatsphäre",
     "section1.title": "Ihre Daten und Nachrichten gehören einzig und allein Ihnen! Hier gehen polyTalk und die DSGVO Hand in Hand!",
     "section1.description": "Vielleicht ist es Ihnen bereits aufgefallen, aber die Privatsphäre der Menschen liegt uns besonders am Herzen. Etwas, das vor allem bei Messenger-Diensten immer wieder thematisiert wird. Wie sicher sind unsere Messenger wirklich? Wir möchten eine echte Alternative schaffen – ohne Kompromisse.\n\npolyTalk ist eine Anwendung innerhalb des polyPod und somit DSGVO-konform per Default. ",
-    "image1": "./images/polyTalk_visual-1.png",
+    "image1": "./images/polyTalk_visual-1.svg",
     "section2.title": "Mehr als nur ein Messenger",
     "section2.description": "Als Teil des polyPod lässt sich polyTalk darüber hinaus auch wunderbar als Kommunikationstool für weitere Anwendungen nutzen, wie beispielsweise innerhalb des polypoly Membership Features oder um in Zukunft das polypoly Support Team zu erreichen.",
-    "image2": "./images/polyTalk_visual-2.png",
+    "image2": "./images/polyTalk_visual-2.svg",
```

Comment:
Well, not these images... It's the same file in the two languages. I don't even think most icons have different language versions.

Reply:
These are not going to be used as icons. Right now they are svgs only because they are simple ones and because we are rushing. In the future you'll probably see other formats. This one in particular has no text and that's why it's the same.

Reply:
To get an idea of the design, you could check [here](https://www.figma.com/file/qSTY8cZ6gbSMx7kPc599fy/MAIN-pod-feature-preview?node-id=2%3A33). Just note that some things have been chaining a lot and maybe they will again.
As for duplicating this images, I don't think it makes sense, we are just increasing the bundle size and not gaining much in return.